vendor/moko/lw004.yaml

```diff
@@ -1,18 +1,31 @@
 name: LW004
-description: The Moko LW004-CT is a wearable tracker and a social safety distance reminder suitable for COVID-19 contact tracing. It can be used to track close contacts by advertising and scanning Bluetooth low-energy beacons. The LW004-CT also includes a 3-axis accelerometer. The scanned data will be sent to the LoRaWAN® network to trace all contacts.
+description: LW004-PB is LoRaWAN based panic button with positioning function, with a compact and smart outlook, and easy to use.
 
 # Hardware versions (optional, use when you have revisions)
 hardwareVersions:
-  - version: '1.0.4'
+  - version: '1.6'
     numeric: 1
 
 # Firmware versions (at least one is mandatory)
 firmwareVersions:
   - # Firmware version
-    version: '2.1.1'
+    version: '3.0.4'
     numeric: 1
     hardwareVersions:
-      - '1.0.4'
+      - '1.4'
+      - '1.5'
+      - '1.6'
+
+    # Firmware features (optional)
+    # Valid values are: remote rejoin (trigger a join from the application layer), transmission interval (configure how
+    # often he device sends a message).
+    features:
+      - remote rejoin
+      - transmission interval
+      - positioning duration
```

Comment:
in this case it only can be `remote join` and `transmission interval` other added features will cause error when you validate